integrations/standalone/tests/pageobjects/Composite.ts

```diff
@@ -60,4 +60,8 @@ export abstract class Composite {
   tags(): any {
     return new Tags(this.page, this.locator);
   }
+
+  currentLocator() {

```

Comment:
locator() does not work because locator is already a property?

Reply:
then it is maybe better to rename the property and make it private. And use locator() outside of the class

Reply:
Yes, and yes. I will change that but I think in a further PR as I have many open PRs which may result in merge conflicts 🙈